### PR TITLE
admin dashboard 

### LINKED ISSUE #32: View Practical Dashboard
As a Faculty Admin, I want to be able to see a summary of activities on my faculty. As seen in https://xd.adobe.com/view/7b24472b-03ee-445f-8d97-6df884bdfc1a-2e77/screen/73e96e66-0c39-4800-b61e-87b494cef1c8.

### LINKED ISSUE #43: Show Course Details
As faculty Admin, I want to be able to see the course detail as shown in https://xd.adobe.com/view/7b24472b-03ee-445f-8d97-6df884bdfc1a-2e77/screen/e478d2a6-5751-45dd-8782-31e68b355e1e.

### PR DESCRIPTION
**Pull request by softdiddy**
courses
created
dashbord-vlab

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -209,13 +212,13 @@ html body{
   color: #00b96b !important;
 }
 .fs1{
-  font-size: 0.9em;
+  font-size: 0.9em !important;
```
Comment (Auwalms) [inline public/css/style.css:215]: @softdiddy can we please stop writing !important everytime.  It makes the code hard to be reused.

File: public/css/style.css [hunk 2]
```diff
@@ -595,7 +597,7 @@ html body{
   position: absolute;
   z-index: 1;
   top: 0%;
-  height: 100vh;
+  /* height: 100vh; */
```
Comment (Auwalms) [inline public/css/style.css]: Why did you comment this out? What happens if the class is reused somewhere? That will break that page.

File: app/Http/Controllers/adminDashboard.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class adminDashboard extends Controller
+{
+    public function index(){
+        return view('admin.dashboard');
```
Comment (Auwalms) [inline app/Http/Controllers/adminDashboard.php]: Is this supposed to be "My Practicals" instead of Dashboard? Please refer to the Design

File: resources/js/components/EnrolledPractical.vue [hunk 4]
```diff
@@ -0,0 +1,382 @@
+<template>
```
Comment (Auwalms) [inline resources/js/components/EnrolledPractical.vue:1]: @softdiddy whats this file for? I thought that was the page that Said worked on last? See this https://github.com/cs-dept-ibbul/vlab/pull/27 and please delete this file from your PR.

File: resources/js/components/PracticalTimeLine.vue [hunk 5]
```diff
@@ -18,21 +18,7 @@
                     <div class="col-md-2">
                       <div class="bttn-container">
                         <button class="btn btn-secondary" type="submit">
-                          <i class="fa fa-plus"></i>Add to Lab
```
Comment (Auwalms) [inline resources/js/components/PracticalTimeLine.vue:21]: @softdiddy can you clarify why this changes? Not sure your current PR has anything to do with this page.

File: resources/views/UserDashboard.blade.php [hunk 6]
```diff
@@ -1,20 +1,195 @@
...
-            <v-lactivity></v-lactivity>
-       
-            <v-practicaltimeline></v-practicaltimeline>
-        
-
-        </div>
-    </div>
-@endsection
\ No newline at end of file
+<div style="display: flex;">
```
Comment (Auwalms) [inline resources/views/UserDashboard.blade.php:46]: @softdiddy Wondering why this changes? Isn't that part of https://github.com/cs-dept-ibbul/vlab/pull/27?

File: resources/views/admin/dashboard.blade.php [hunk 7]
```diff
@@ -0,0 +1,102 @@
+@extends('../layouts/main')
+@section('content-body')
+<div style="display: flex;">   
+	<!-- main side bar -->
+   <v-msidebar home="{{route('home')}}" explore="{{route('explore')}}" settings="{{route('home')}}" mycourse="{{route('my-course-review')}}" courses="{{route('courses')}}" incourse="1" active="courses"  ></v-msidebar>   
+	<!-- end main side bar -->
+
+   <div class="expr-dashboard-container" style="background-color:#eee;">
+   		<!-- top nav bar -->
+        <v-systopnave title="Dashboard" username='said'></v-systopnave>
```
Comment (Auwalms) [inline resources/views/admin/dashboard.blade.php]: Please update the title to My Practicals just  as it is on the design @softdiddy

File: routes/web.php [hunk 8]
```diff
@@ -20,7 +20,8 @@
 Route::get('/my-courses', 'App\Http\Controllers\StudentCourses@index')->name('courses');
 Route::get('/my-course-review/{id?}', 'App\Http\Controllers\StudentCourses@review')->name('my-course-review')->where('id', '[0-9]+');;
 //{name?}
-
+//AMDIN
+Route::get('/dashboard', 'App\Http\Controllers\adminDashboard@index')->name('dashboard');
```
Comment (Auwalms) [inline routes/web.php]: Whats this route for? I see you added the correct route in line 54 already

**Review by Auwalms — CHANGES_REQUESTED** [review]
Please see comments

File: public/css/style.css (hunk 1 above)
Comment (softdiddy) [reply public/css/style.css:215]: yeah, its was created by said so that we can apply it where we need it

File: public/css/style.css (hunk 2 above)
Comment (softdiddy) [reply public/css/style.css]: we do not use it any more

File: public/css/style.css (hunk 1 above)
Comment (Auwalms) [reply public/css/style.css:215]: I understand, We have agreed to stop using `!important` already

Comment (saidabdul80) [reply public/css/style.css:215]: If u don't need it don't use the class. However it doesn't touch other or override other part except you use the same class name and this is the importance of important, in this case my work will not be affected now is left for u to find out that the class name is used. Note: it is not use on a tag @Auwalms

File: app/Http/Controllers/adminDashboard.php (hunk 3 above)
Comment (softdiddy) [reply app/Http/Controllers/adminDashboard.php]: yes, corrected

File: public/css/style.css (hunk 1 above)
Comment (Auwalms) [reply public/css/style.css:215]: Yes @saidabdul80, I didnt see a good reason why he added !important since its not going to clash with any class. @softdiddy please remove the !important you added here.

Comment (saidabdul80) [reply public/css/style.css:215]: and also Note: it is used on a class with just one or two properties not class with many property which You might need to modify one of it to suit your work. it 💯 not affecting any one's work.

Comment (saidabdul80) [reply public/css/style.css:215]: The reason of use is to avoid override sir, let assume someone else in the team wrote a code that can override that property while he intend to use it somewhere else @Auwalms  please refer to the message before this and understand sir

File: resources/js/components/PracticalTimeLine.vue (hunk 5 above)
Comment (softdiddy) [reply resources/js/components/PracticalTimeLine.vue:21]: no changes sir

File: resources/views/UserDashboard.blade.php (hunk 6 above)
Comment (softdiddy) [reply resources/views/UserDashboard.blade.php:46]: No changes, may be its just space or something

File: resources/views/admin/dashboard.blade.php (hunk 7 above)
Comment (softdiddy) [reply resources/views/admin/dashboard.blade.php]: done

File: routes/web.php (hunk 8 above)
Comment (softdiddy) [reply routes/web.php]: corrected, its suppose to be "my-practicals"

File: public/css/style.css (hunk 1 above)
Comment (Auwalms) [reply public/css/style.css:215]: We will review and tell the person na. Its the reason why we said no !important

File: resources/js/components/PracticalTimeLine.vue (hunk 5 above)
Comment (Auwalms) [reply resources/js/components/PracticalTimeLine.vue:21]: But it looks like you deleted the from line 21?

Comment (softdiddy) [reply resources/js/components/PracticalTimeLine.vue:21]: No, still there

**Review by Auwalms — APPROVED** [review]

**Comment by saidabdul80** [comment]
Nice one sir

**Review by saidabdul80 — APPROVED** [review]
